### PR TITLE
Try to print login page source code.

### PR DESCRIPTION
**Pull request by han-gao**
Try to fix [ondemandIT error](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=939068&view=logs&j=8b2916d1-c2a9-5ac5-75e7-bb27ae19f1fa&t=fc88adc9-cbe0-5401-701f-ab9b14bd1cfe&s=5845226e-7044-5fd2-a201-bac5ecdb063f).

### CODE REVIEW

File: sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/AADSeleniumITHelper.java [hunk 1]
```diff
@@ -49,8 +49,14 @@ public AADSeleniumITHelper(Class<?> appClass, Map<String, String> properties, St
...
     public void logIn() {
-        driver.get(app.root() + "oauth2/authorization/azure");
+        String loginUrl = app.root() + "oauth2/authorization/azure";
+        driver.get(loginUrl);
+        LOGGER.info(loginUrl);
+        String BeforeLoginPageSource = driver.getPageSource();
+        LOGGER.info(BeforeLoginPageSource);
         wait.until(ExpectedConditions.visibilityOfElementLocated(By.name("loginfmt"))).sendKeys(username + Keys.ENTER);
+        String AfterLoginPageSource = driver.getPageSource();
+        LOGGER.info(AfterLoginPageSource);
```
Comment (stliu) [inline sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/AADSeleniumITHelper.java:59]: dont think this should be info level

File: sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/AADSeleniumITHelper.java [hunk 2]
```diff
@@ -49,8 +49,14 @@ public AADSeleniumITHelper(Class<?> appClass, Map<String, String> properties, St
     }
 
     public void logIn() {
-        driver.get(app.root() + "oauth2/authorization/azure");
+        String loginUrl = app.root() + "oauth2/authorization/azure";
+        driver.get(loginUrl);
+        LOGGER.info(loginUrl);
```
Comment (backwind1233) [inline sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/AADSeleniumITHelper.java:54]:
```
driver.get(loginUrl);
LOGGER.info(loginUrl);
```
I think we can change to 
```
LOGGER.info(loginUrl);
driver.get(loginUrl); 
```

File: sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/AADSeleniumITHelper.java [hunk 3]
```diff
@@ -49,8 +49,14 @@ public AADSeleniumITHelper(Class<?> appClass, Map<String, String> properties, St
...
+        String BeforeLoginPageSource = driver.getPageSource();
```
Comment (backwind1233) [inline sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/AADSeleniumITHelper.java:55]: I suggest the variable name begin with lower case;
```
BeforeLoginPageSource --> beforeLoginPageSource
AfterLoginPageSource   --> afterLoginPageSource
```

**Pull request closed without merging**